Unity2DPlatformer/ProjectSettings/TagManager.asset

```diff
@@ -10,6 +10,8 @@ TagManager:
   - Spikes_Vert_R
   - Spikes_Hor_D
   - Spikes_Hor_U
+  - EndGameChest
+  - spikes
```

Comment:
Good idea with spikes, there could be invisible wall around level 👍+1
But I think that for chest there should be created prefab as there is for other collectables